opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -219,9 +219,14 @@ def create_time_series_list(self, v_data, option_resource_type,
                 # point.value.distribution_value.range.min = agg_data.min
                 # point.value.distribution_value.range.max = agg_data.max
                 bounds = dist_value.bucket_options.explicit_buckets.bounds
+                # Stackdriver expects an "underflow" bucket for samples in
```

Comment:
"The first bucket bound should be 0.0 because the Metrics first bucket is [0, first_bound) but Stackdriver monitoring bucket bounds begin with -infinity (first bucket is (-infinity, 0))." I would use this.